docs/design.md

```diff
@@ -28,6 +28,7 @@ There is a more detailed explanation in the following paragraphs.
 | *Name* field in namespace | *Name* gets set to 'pmem-csi' to achieve own vs. foreign marking | *Name* gets set to VolumeID, without attempting own vs. foreign marking  |
 |Minimum volume size| 4 MB                   | 1 GB (see also alignment adjustment below) |
 |Alignment requirements |LVM creation aligns size up to next 4MB boundary  |driver aligns  size up to next alignment boundary. The default alignment step is 1 GB. Device(s) in interleaved mode will require larger minimum as size has to be at least one alignment step. The possibly bigger alignment step is calculated as interleave-set-size multiplied by 1 GB |
+|Huge pages supported<sup>2</sup> | no| yes|
```

Comment:
I just noticed that the footnote number doesn't match the new footnote. This needs to be 4, not 2.

Also, can you replace "no" with "maybe" to match our current understanding of the problem?